script.js

```
@@ -0,0 +1,405 @@
+                    `Failed to insert ${value}`;
+                break;
+            
+            case 'delete':
+                success = this.linkedList.delete(value);
+                message = success ? 
+                    `Successfully deleted ${value} from the linked list!` : 
+                    `Value ${value} not found in the linked list`;
+                break;
+            
+            case 'search':
+                const found = this.linkedList.search(value);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)